go/tasks/plugins/hive/execution_state.go

```diff
@@ -334,15 +335,20 @@ func KickOffQuery(ctx context.Context, tCtx core.TaskExecutionContext, currentSt
 		return currentState, errors.Wrapf(errors.RuntimeFailure, err, "Failed to read token from secrets manager")
 	}
 
-	query, clusterLabelOverride, tags, timeoutSec, err := GetQueryInfo(ctx, tCtx)
+	query, clusterLabelOverride, tags, timeoutSec, taskName, err := GetQueryInfo(ctx, tCtx)
 	if err != nil {
 		return currentState, err
 	}
 
 	clusterPrimaryLabel := getClusterPrimaryLabel(ctx, tCtx, clusterLabelOverride)
 
+	taskExecutionIdentifier := tCtx.TaskExecutionMetadata().GetTaskExecutionID().GetID()
+	commandMetadata := client.CommandMetadata{TaskName: taskName,
+		Domain:  taskExecutionIdentifier.TaskId.Domain,
```

Comment:
I usually like to stick with `Get` or not the whole way through.  I guess it doesn't really matter though.

Reply:
Fixed.